pyccel/codegen/printing/ccode.py

```diff
@@ -952,7 +952,8 @@ def _print_Import(self, expr):
         if expr.target:
            dtype = expr.target.pop().name
            if source.startswith('stc/'):
-             import_file,_ = source.split('_')
+             import_file, *_ = source.split('_')
+             _ = '_'.join(_)
```

Comment:
Another option is to use the `maxsplit` argument of `split`. This avoids unnecessary calculations